program/src/main/ExamImage.vala

```diff
@@ -367,111 +337,99 @@ public class ExamImage
                     }
                     break;
                 }
+
+                case Gdk.Key.n: 

```

Comment:
Because these cases are so long, if possible, they should be put in function instead

Reply:
This is a good idea also since I should eventually add GUI buttons to execute these procedures as well so that the user isn't forced into using these hotkeys. I've moved these to exit_setup() and update_bounds() respectively, which has shortened the cases by a bit. I've left some of the conditional statements that check and prevent error  cases within the cases though, since I believe it's not within scope/concern of the actual procedure that the functions are doing.